### PR TITLE
Issue 12

### PR DESCRIPTION
**Pull request by sourrust**
Fixes for issue #12
- jim mappings for arrow keys, `Delete`, and `Backspace`
- also included mappings for `Space`

### CODE REVIEW

File: src/commands.coffee [hunk 1]
```diff
@@ -256,11 +256,23 @@ map 'u', class Undo extends Command
 # Delete the char under the cursor.
 map 'x', class DeleteChar extends Command
   exec: (jim) -> new Delete(1, new MoveRight @count).exec jim
-  
+  visualExec: (jim) -> Delete::visualExec jim
```
Comment (misfo) [inline src/commands.coffee:259]: Oooh. Bonus feature!

File: src/commands.coffee [hunk 2]
```diff
@@ -256,11 +256,23 @@ map 'u', class Undo extends Command
...
+
 # Delete the char before the cursor.
 map 'X', class Backspace extends Command
   exec: (jim) -> new Delete(1, new MoveLeft @count).exec jim
+  visualExec: (jim) ->
+    Delete::linewise = true
```
Comment (misfo) [inline src/commands.coffee]: Oops. We don't want do change `Delete`'s prototype here... Wanna just instantiate one?

**Comment by misfo** [comment]
Look pretty good, thanks!  I'll test this out later and get it merged in

File: src/commands.coffee (hunk 2 above)
Comment (sourrust) [inline src/commands.coffee]: Yeah, I did. Didn't notice the effect it would have until now.

**Comment by misfo** [comment]
All merged in. Thanks, @sourrust!

**Pull request merged**